packages/app-apw/src/hooks/useFetchInterval.ts

```diff
@@ -0,0 +1,77 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+
+interface UseFetchDataProps {
+    callback: () => Promise<any>;
+    interval: number;
+    loading: boolean;
+}
+
+interface UseFetchDataResult {
+    error: Record<string, any> | null;
+}
+
+export const useFetchInterval = ({
+    callback,
+    interval,
+    loading
+}: UseFetchDataProps): UseFetchDataResult => {
+    const [error, setError] = useState<Record<string, any> | null>(null);
+    const timer = useRef<string | number | NodeJS.Timeout | undefined>(undefined);
+
+    const [initiallyLoaded, setInitiallyLoaded] = useState<boolean>(false);
```

Comment:
@brunozoric I have one fix on your code. I removed the `undefined` and set `false` value to `initiallyLoaded`. 

The problem was with the `loading` state in the `useContentReviewr` hook, first returned value was `false`. 
The content review steps didn't receive any updates.

Now I tested all cases everything works fine.